backend/src/kitconcept/intranet/distributions/intranet/content/content/195d96ca9952493cbffa69cda040b8b3/data.json

```
@@ -27,7 +27,7 @@
   "modified": "2025-07-25T15:25:07+00:00",
   "nav_title": null,
   "preview_caption_link": null,
-  "remoteUrl": "/features/content-types",
+  "remoteUrl": "http://nohost/Plone/features/content-types",
```

Fixing this manually so the rest of the PR can be merged.

```suggestion
  "remoteUrl": "/features/content-types",
```